partials/txListType/payingForTx.vue

```diff
@@ -0,0 +1,171 @@
+          <AppDefinition
+            v-if="transaction.tx.fee"
+            title="tx fee"
+          >
+            <FormatAeUnit :value="transaction.tx.fee" />
+          </AppDefinition>
+          <AppDefinition
+            v-if="transaction.tx.fee || transaction._status == 'mined'"
+            title="total cost"
+          >
+            <span v-if="transaction._status == 'mined'">
+              <app-loading tooltip-text="Waiting for mdw sync" />
+            </span>
+            <FormatAeUnit
+              v-else
+              :value="transaction.tx.fee"
+            />
+          </AppDefinition>
+          <AppDefinition
+            v-if="transaction.tx.nonce"
+            title="nonce"
+          >
+            {{ transaction.tx.nonce }}
+          </AppDefinition>
+          <AppDefinition
+            v-if="transaction.microTime"
+            title="Time"
+            class="tx-time"
+          >
+            {{ transaction.microTime | timestampToUTC }}
+          </AppDefinition>
+        </div>
+      </div>
+    </div>
+    <GaAttachTx
+      v-if="transaction.tx.tx && transaction.tx.tx.tx.type == 'GAAttachTx'"
+      :transaction="{
+        ...transaction,
+        tx: {
+          ...transaction.tx.tx.tx
+        }
+      }"
+    />
+  </div>
+</template>
+<script>
+import GaAttachTx from './gaAttachTx'
+import AppDefinition from '../../components/appDefinition'
+import FormatAeUnit from '../../components/formatAeUnit'
+import Account from '../../components/account'
+import LabelType from '../../components/labelType'
+import timestampToUTC from '../../plugins/filters/timestampToUTC'
+import formatToken from '../../plugins/filters/formatToken'
+import { transformTxType } from '../../store/utils'
+import AppLoading from '../../components/appLoading.vue'
+
+export default {
+  name: 'PayingForTx',
+  components: {
+    GaAttachTx,
+    LabelType,
+    AppDefinition,
+    FormatAeUnit,
+    Account,
+    AppLoading
+  },
+  filters: {
+    timestampToUTC,
+    transformTxType,
+    formatToken
+  },
+  props: {
+    transaction: {
+      type: Object,
+      required: true
+    }
+  },
+  methods: {
+    getTitle (transaction) {
+      if (transaction.tx.function) {
+        if (transaction.tx.function.includes('swap')) {
+          return 'Swap'
+        }
+
+        return transaction.tx.function.replaceAll('_ae', '').replaceAll('_', ' ')
+      }
+      return 'token transfer'
+    },
+    getAmount (token) {
+      return formatToken(token.amount, token.decimals, token.symbol)
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "~@aeternity/aepp-components-3/src/styles/variables/colors";
+.status {
+  text-transform: uppercase;
+  &.ok {
+    color: $color-alternative;
+  }
+  &.revert {
+    color: $color-red;
+  }
+}
+.contract-call.transfer {
+  border-left: 2px #14CCB7 solid;
+
+  .transaction:nth-child(2) .transaction-main-info .transaction-main-info-inner .label-type {
```

Comment:
Just a side question - we are not using any class name methodology - like BEM?
Because I would not advise dependent selectors

Reply:
I just followed what is been used previously in the other components.
in the new explorer version we will define a better approach for styling